src/runtime/pkg/katautils/config.go

```diff
@@ -386,12 +386,32 @@ func (h hypervisor) GetEntropySource() string {
 	return h.EntropySource
 }
 
+var procCPUInfo = "/proc/cpuinfo"
+
+func getHostCPUs() uint32 {
```

Comment:
Would it be better to return the value as the lesser of the two numbers: one from `/proc/cpuinfo` and the other from `NumCPU()`?

Reply:
No, I don’t think that’s what we want. Pls read commit message which explains rationale for current pattern.